# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1709828317 -3600
# Parent  0c8c725d014f7fb356d9eaf07344eb33e4153e30
Feature: Set icon on more stand-alone windows.

diff --git a/mail/base/content/FilterListDialog.xhtml b/mail/base/content/FilterListDialog.xhtml
--- a/mail/base/content/FilterListDialog.xhtml
+++ b/mail/base/content/FilterListDialog.xhtml
@@ -17,8 +17,9 @@
 ]>
 <html id="filterListDialog" xmlns="http://www.w3.org/1999/xhtml"
       xmlns:xul="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
       xmlns:html="http://www.w3.org/1999/xhtml"
+      icon="messengerWindow"
       windowtype="mailnews:filterlist"
       lightweightthemes="true"
       persist="width height screenX screenY"
       scrolling="false"
diff --git a/mail/base/content/SearchDialog.xhtml b/mail/base/content/SearchDialog.xhtml
--- a/mail/base/content/SearchDialog.xhtml
+++ b/mail/base/content/SearchDialog.xhtml
@@ -24,8 +24,9 @@
 ]>
 <html id="searchMailWindow" xmlns="http://www.w3.org/1999/xhtml"
       xmlns:xul="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
       xmlns:html="http://www.w3.org/1999/xhtml"
+      icon="messengerWindow"
       windowtype="mailnews:search"
       scrolling="false"
       style="min-width:52em; min-height:34em;"
       lightweightthemes="true"
diff --git a/mail/base/content/viewSource.xhtml b/mail/base/content/viewSource.xhtml
--- a/mail/base/content/viewSource.xhtml
+++ b/mail/base/content/viewSource.xhtml
@@ -18,8 +18,9 @@
 
 <window id="viewSource"
         xmlns="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
         xmlns:html="http://www.w3.org/1999/xhtml"
+        icon="messengerWindow"
         contenttitlesetting="true"
         title="&mainWindow.title;"
         titlemodifier="&mainWindow.titlemodifier;"
         titlepreface="&mainWindow.preface;"
diff --git a/mail/components/addrbook/content/abSearchDialog.xhtml b/mail/components/addrbook/content/abSearchDialog.xhtml
--- a/mail/components/addrbook/content/abSearchDialog.xhtml
+++ b/mail/components/addrbook/content/abSearchDialog.xhtml
@@ -19,8 +19,9 @@
 ]>
 <window id="searchAddressBookWindow"
         xmlns="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
         xmlns:html="http://www.w3.org/1999/xhtml"
+        icon="messengerWindow"
         onload="searchOnLoad();"
         onunload="searchOnUnload();"
         onclose="onSearchStop();"
         windowtype="mailnews:absearch"
diff --git a/mail/extensions/openpgp/content/ui/enigmailKeyManager.xhtml b/mail/extensions/openpgp/content/ui/enigmailKeyManager.xhtml
--- a/mail/extensions/openpgp/content/ui/enigmailKeyManager.xhtml
+++ b/mail/extensions/openpgp/content/ui/enigmailKeyManager.xhtml
@@ -16,8 +16,9 @@
   id="enigmailKeyManager"
   data-l10n-id="openpgp-key-man-title"
   xmlns="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
   xmlns:html="http://www.w3.org/1999/xhtml"
+  icon="messengerWindow"
   lightweightthemes="true"
   onload="enigmailKeyManagerLoad();"
   height="450"
   width="700"
diff --git a/mailnews/extensions/newsblog/feed-subscriptions.xhtml b/mailnews/extensions/newsblog/feed-subscriptions.xhtml
--- a/mailnews/extensions/newsblog/feed-subscriptions.xhtml
+++ b/mailnews/extensions/newsblog/feed-subscriptions.xhtml
@@ -22,8 +22,9 @@
   id="feedSubscriptions"
   xmlns="http://www.w3.org/1999/xhtml"
   xmlns:xul="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
   xmlns:html="http://www.w3.org/1999/xhtml"
+  icon="messengerWindow"
   scrolling="false"
   windowtype="Mail:News-BlogSubscriptions"
   persist="width height screenX screenY sizemode"
   lightweightthemes="true"
